src/borg/helpers.py

```diff
@@ -1404,12 +1417,31 @@ def __init__(self):
             if self.logger.level == logging.NOTSET:
                 self.logger.setLevel(logging.WARN)
             self.logger.propagate = False
+        self.emit = self.logger.getEffectiveLevel() == logging.INFO
 
     def __del__(self):
         if self.handler is not None:
             self.logger.removeHandler(self.handler)
             self.handler.close()
 
+    def output_json(self, *, finished=False, **kwargs):
+        assert self.json
+        if not self.emit:
+            return
+        kwargs.update(dict(
+            operation=self.id,
+            msgid=self.msgid,
+            type=self.JSON_TYPE,
+            finished=finished
+        ))
+        print(json.dumps(kwargs), file=sys.stderr)
```

Comment:
hmm, is this making the assumption that output shall always be on stderr?
the logging system can log to many different outputs.